Modules/IO/NIFTI/include/itkNiftiImageIO.h

```diff
@@ -27,6 +27,30 @@
 
 namespace itk
 {
+
+
+/** \class Analyze75Flavour
+ * \ingroup ITKIONIFTI
+ * Enum used to define way to treat legacy Analyze75 files
+ */
+enum class Analyze75Flavour : uint8_t
+{
+  /** Behavioir introduced in ITK4.0 by NIFTI reader interpreting Analyze files */
+  AnalyzeITK4 = 4,
+  /** Will ignore orientation code and negative pixel dimensios */
+  AnalyzeFSL = 3,
+  /** Will ignore orientation code and respect negative pixel dimensios */
+  AnalyzeSPM = 2,
+  /** Same as AnalyzeITK4 but will show warning about deprecated file format (Default)*/
+  AnalyzeITK4Warning = 1,
+  /** Reject Analyze files as potentially wrong  */
+  AnalyzeReject = 0
+};
+
```

Comment:
Couple of typos: Behavioir, dimensios

Can you add a comment indicating which behaves like ITK 3/4 AnalyzeImageIO.

Reply:
It's a tricky question: ITK3 allowed negative spacing, the most similar behavior of Analyze reader would be SPM